src/pages/JobSearch/index.jsx

```diff
@@ -60,6 +60,11 @@ function JobSearch() {
   // This is used to determine next result items to show when the user clicks "Load More" button.
   const [cursor, setCursor] = useState(12);
   const [jobs, setJobs] = useState(jobsData.slice(0, cursor));
+  const [filteredJobs, setFilteredJobs] = useState(jobsData);
+
+  const handleSearch = (filteredJobs) => {
+    setFilteredJobs(filteredJobs);
```

Comment:
Same note as above regarding the use of `jobsData` vs `jobs`.

Reply:
Updated the component to use `jobs` instead of `jobsData`